app/communication/views/banner.py

```diff
@@ -13,9 +13,13 @@
 class BannerViewSet(BaseViewSet):
     serializer_class = BannerSerializer
     pagination_class = BasePagination
-    queryset = Banner.objects.all()
+    queryset = Banner.objects.order_by("visible_from")
```

Comment:
Ikke gjør dette er du grei. Bruk heller django filter og muligheten for frontend å velge ordering.

Reply:
Ser også at dette er feil endring for hva pr sier.

Reply:
Ja whoops, innser at jeg glemte å lage ny branch fra main.